tests/vertical_playground.html

```diff
@@ -482,6 +496,40 @@ <h1>Vertical Blocks</h1>
           <option value="top">Top</option>
           <option value="bottom">Bottom</option>
         </select>
+        <select name="locale" onchange="setLocale(this.value)">
```

Comment:
maybe eventually this list should be generated based on what's in `msg/js` or `msg/json`

Reply:
I agree that would be nice to do someday.